docs/PLUGINS.md

```diff
@@ -289,24 +289,25 @@ You also can write a test case to check export from a live instance is well supp
     }
 ```
 
-_Hint:  use a class rule if you just want to test a single import and export_
+The helper methods shown are provided by the `io.jenkins.plugins.casc.misc.Util` class.
```

Comment:
We could switch to having the imports as part of the snippet to avoid this comment

Reply:
I also considered having the imports in the example but as @timja stated, the snippet looks a bit bloated then. If I just add the imports it would look like this which I don't really like:

```java
...
import static io.jenkins.plugins.casc.misc.Util.getUnclassifiedRoot;
import static io.jenkins.plugins.casc.misc.Util.toStringFromYamlFile;
import static io.jenkins.plugins.casc.misc.Util.toYamlString;
...

    @Test
    @ConfiguredWithCode("configuration-as-code.yml")
    public void should_support_configuration_export() throws Exception {
        ConfiguratorRegistry registry = ConfiguratorRegistry.get();
        ConfigurationContext context = new ConfigurationContext(registry);
        CNode yourAttribute = getUnclassifiedRoot(context).get("<your-attribute>");

        String exported = toYamlString(yourAttribute);

        String expected = toStringFromYamlFile(this, "expected_output.yaml");

        assertThat(exported, is(expected));
    }
```
Therefore I would still go for current change, using an sentence for explanation.

Reply:
Seems reasonable 👍